pyproject.toml

```diff
@@ -28,7 +28,7 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.6,<=3.9"
+python = ">=3.6,<3.10"
```

Comment:
Restricting python version to `<=3.9` does not allow minor updates (like `3.9.2` I currently use 😅)